docs/snapshotting/snapshot-support.md

```diff
@@ -9,6 +9,10 @@ guest workload at that particular point in time.
 
 ## Snapshotting in Firecracker
 
+### Supported platforms
+Snapshotting is supported on Intel Skylake, Cascade Lake and AMD Zen2 CPUs. These are the officially supported CPUs, but the functionality *may* work on other CPUs with hardware virtualization support. Our plan is to incrementally extend support for other platforms and CPUs.
```

Comment:
could we say something like "snapshotting is supported on all General availability targets supported by Firecracker"?

Reply:
Updated.